bundles/statistics/statsgrid/view/Diagram/Diagram.jsx

```diff
@@ -342,16 +343,22 @@ export const Diagram = ({ chartData, sortOrder }) => {
 
     return (
         <Content>
-            <div
-                ref={labelsRef}
-                className='statsgrid-labels'
-            />
-            <Chart>
+            {!chartData?.data ? (
```

Comment:
We should just have another return statement rendering the no data message. It makes it easier to read than having the multiline if-statement in the middle of the JSX.